libdnf/libdnf5.pc.in

```diff
@@ -2,7 +2,7 @@ prefix=@CMAKE_INSTALL_PREFIX@
 libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
 includedir=${prefix}/include
 
-Name: libdnf
+Name: libdnf5
```

Comment:
Shouldn't also `Libs: -L${libdir} -ldnf` in this file be changed to `Libs: -L${libdir} -ldnf5`?
(Sorry, github doesn't allow me to comment on the more appropriate line #10)

Reply:
Updated, @m-blaha May I ask you to look at the last update?